docs/reference/UDS Core/functional-layers.md

```diff
@@ -46,3 +46,41 @@ packages:
     repository: ghcr.io/defenseunicorns/packages/uds/core-backup-restore
     ref: 0.29.0-upstream
 ```
+
+## Layer Selection
+
+Layer selection will always be deployment-specific but below are guidelines for what layers to consider for your deployment.  The layers marked with a four pointed star (🟄) are those needed to follow the [Big Bang Conformant Stack](https://repo1.dso.mil/big-bang/product/bbtoc/-/blob/master/policy/conformance.md?ref_type=heads) though if you are not bound by that document, UDS Core Base is the only *technical* layer required to install most UDS Packages.
```

Comment:
Can we pick a different symbol here (for the four pointed star)? On GitHub and on the docs site this symbol shows as 🟄 (included screenshot in case this is machine/font specific).

![Screenshot 2025-03-21 at 1 50 18 PM](https://github.com/user-attachments/assets/1901ea36-5f66-496c-b9c7-1ce945ea1c93)